.github/workflows/ci_pipeline.yml

```diff
@@ -395,13 +356,13 @@ jobs:
 #
 #    steps:
 #      - name: Checkout
-#        uses: actions/checkout@v3
```

Comment:
Do we need this entire job anymore? I guess this is made redundant by the actually working implementation of this. Should we delete the comments?

Reply:
From my point of view, I can also remove that job and the other one, that is commented out. I just didn't want to do that myself, as I was not sure if you (whoever) wanted to reactivate them in the future. But I can also happily remove them if that is fine with everyone.

Reply:
Removed jobs in 48d7d4bbe34c5375ddfb981ee1b711c24a91f54c